__tests__/app.test.js

```diff
@@ -232,3 +232,48 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("returns relevant comments for specific article ID", () => {
+    return request(app)
+      .get("/api/articles/5/comments")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.comments.length).toBe(2);
+        expect(body.comments[0]).toEqual(
+          expect.objectContaining({
+            comment_id: 14,
+            body: "What do you see? I have no idea where this will lead us. This place I speak of, is known as the Black Lodge.",
+            article_id: 5,
+            author: "icellusedkars",
+            votes: 16,
+          })
+        );
+        expect(body.comments[1]).toEqual(
+          expect.objectContaining({
+            comment_id: 15,
+            body: "I am 100% sure that we're not completely sure.",
+            article_id: 5,
+            author: "butter_bridge",
+            votes: 1,
+          })
+        );
+      });
+  });
+  test("If an article has no comments, returns 404 error", () => {
+    return request(app)
+      .get("/api/articles/2/comments")
+      .expect(404)
```

Comment:
This article exists but it just so happens that it has no comments!

A `404` would be misleading and might cause users to think the article didn't exist

This should be a `200` returning an empty array!